src/akushon/action/node/action_manager.cpp

```diff
@@ -90,6 +103,25 @@ void ActionManager::load_config(const std::string & path)
   }
 }
 
+void ActionManager::set_config(const nlohmann::json & json)
+{
+  for (auto &[key, val] : json.items()) {
+    if (key == "dynamic_kick") {
```

Comment:
Should use [`nlohmann::json::find`](https://json.nlohmann.me/api/basic_json/find/) instead of for-looping the items.